src/pipx/commands/reinstall.py

```diff
@@ -28,10 +28,14 @@ def reinstall(
         return EXIT_CODE_REINSTALL_VENV_NONEXISTENT
 
     try:
+        # use PurePath.relative_to in a try block instead
+        # of PurePath.is_relative_to, for python 3.6-3.8 compatability
         Path(python).relative_to(venv_dir)
+        python_relative_to_venv_dir = True
     except ValueError:
-        pass
-    else:
+        python_relative_to_venv_dir = False
+
+    if python_relative_to_venv_dir:
```

Comment:
This change seems unrelated, what happened here?

Reply:
I've reverted the PR to avoid merging this. Please open a new PR and keep the change set scopped to the PR description.